### ISSUE TITLE
color headline view text a specific way for battle.net friends

### ISSUE THREAD

**Issue by ANaPhOrAX** [Luxocracy/NeatPlates#440]
**Is your feature request related to a problem? Please describe.**
Calling it a problem would be a bit much, but it's something that is missing anyway.

**Describe the solution you'd like**
There's currently afaik no option to, for headline view, have the text be colored a specific way for people who are on your battle.net friend list. You can color them for friendly, party, guild members, but not battle.net friends. This does happen with default wow names though. Was wondering if you could implement this, it'd be freakin' sexy. Like yo, don't these bright blue names look beautiful? https://i.imgur.com/qYYdy.jpeg I want that beauty back :D 

**Describe alternatives you've considered**
Can't think of any tbh. I guess not using neatplates.. but that's not really a serious option for me tbh.

**Comment by Luxocracy** [comment]
~~There should already be a special color for this, however, it uses the same color you have set for your guild members.~~
Actually, this isn't functioning. Assuming it was removed a long while ago when battle.net was introduced and never re-added.

I've fixed this in the latest release so it should be working, but it still uses the same color as guild members. I could add a separate color option for this if that's necessary.

**Comment by ANaPhOrAX** [comment]
> I could add a separate color option for this if that's necessary.

yes please! if you could, I'd really appreciate it. I can imagine someone with a lot of overlap between guild/friends wouldn't care, 
but I'm not particularly close to my guild, so I'd definitely like to seperate them, color-wise

**Comment by ANaPhOrAX** [comment]
oh I just realized something else btw. the names turn green in pvp. any chance that while you're at it you could also include an option for headline view name color for pvp environments?

because atm it's a choice between having the in party headline view name color match the blizz party name color (blue/purple) or the pvp group color (green), or something else entirely. so there's currently always at least one scenario where the color changes depending on how far away you are. if you set it to the blue/purple blizz has for parties it's nice outside of pvp, but when you do a bg it's just weird when from a distance a player's name is green and then when u get close and it switches to said blue/purple and vice/versa

**Comment by ANaPhOrAX** [comment]
one last thing I noticed btw: guild name color (as well as bnet friend color) should supercede party/pvp group color. should as in, that's how it works by default (see screenshot I linked in OP). atm it doesn't do that. was in a group with a bnet friend, and their name was colored the same as everyone else's in the group.

**Comment by Luxocracy** [comment]
I believe the order should currently be 
Guild member -> Friend -> Party member

That is as long as you have your name coloring set to "By Reaction". As otherwise they won't have any effect.

One could argue that Friend should override Guild member as they now have separate colors, but I don't really see a way that group color would override the

**Comment by ANaPhOrAX** [comment]
hm weird cause I do have it set to by reaction, but friend/guild color isn't coming before party color atm.

and yeah, for blizz name colors, friend does come before guild. I also know pvp comes before party. so if you're in a regular party but someone is flagged for pvp, their name will be green. I think the order is friend > guild > pvp > party.

ill do some testing and report back
